libtlafmt/src/ast_format/node.rs

```diff
@@ -682,6 +682,55 @@ X == A \/ B
 ---- MODULE B ----
 X == "bananas"
 ====
+"#
+        );
+    }
+
+    #[test]
+    fn test_spec_always_true_ident() {
+        assert_rewrite!(
+            r#"
+---- MODULE B ----
+LEMMA TypeCorrect == Spec => []TypeInv
+THEOREM DeadlockFreedom == Spec => [] Invariant
+====
+"#
+        );
+    }
+
+    /// Reproducer for https://github.com/domodwyer/tlafmt/issues/25.
+    #[test]
+    fn test_bounded_quantification() {
```

Comment:
This is an isolated test case for the input that was mishandled such that the formatted result was an invalid spec.